[bug] Missing implementation for the char data type

The compiler recognized these data-types, however, the eo-runtime does not have the corresponding objects for them. Hence, the following errors are produced on compilation.
And for `char`:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project quick-start: Compilation failure: Compilation failure: 
[ERROR] /home/nlchar/sandbox/target/generated-sources/eo/EOapp.java:[30,39] cannot find symbol
[ERROR]   symbol:   class EOchar
[ERROR]   location: package org.eolang
[ERROR] /home/nlchar/sandbox/target/generated-sources/eo/EOapp.java:[32,68] cannot find symbol
[ERROR]   symbol:   variable a
[ERROR]   location: class sandbox.EOapp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```
***The syntax for `char`:***

![CHAR](https://user-images.githubusercontent.com/64870515/105915785-2aac2e80-6041-11eb-9e22-70e9aeae9ac8.png)
  
**EBNF:**
```EBNF
CHAR     ::= "'" [0-9a-zA-Z] "'"
```

--- a/eo-maven-plugin/src/main/resources/org/eolang/maven/pre/data.xsl
+++ b/eo-maven-plugin/src/main/resources/org/eolang/maven/pre/data.xsl
@@ -35,6 +35,11 @@ SOFTWARE.
             </xsl:when>
           </xsl:choose>
           <xsl:choose>
+            <xsl:when test="@data='char'">
+              <xsl:text>Character</xsl:text>
+            </xsl:when>
+          </xsl:choose>
+          <xsl:choose>
             <xsl:when test="@data='float'">
               <xsl:text>Double</xsl:text>
             </xsl:when>
@@ -55,6 +60,11 @@ SOFTWARE.
             <xsl:text>"</xsl:text>
             <xsl:value-of select="text()"/>
             <xsl:text>"</xsl:text>
+          </xsl:when>
+          <xsl:when test="@data='char'">
+            <xsl:text>'</xsl:text>
+            <xsl:value-of select="text()"/>
+            <xsl:text>'</xsl:text>
           </xsl:when>
           <xsl:when test="@data='int'">
             <xsl:value-of select="text()"/>

--- a/eo-runtime/src/main/eo/org/eolang/char.eo
+++ b/eo-runtime/src/main/eo/org/eolang/char.eo
@@ -1,0 +1,7 @@
++package org.eolang
++alias sprintf org.eolang.txt.sprintf
+
+[data] > char
+  # Converts char to string
+  [] > toString
+    sprintf "%c" data > @

--- a/eo-runtime/src/main/eo/org/eolang/char.eo
+++ b/eo-runtime/src/main/eo/org/eolang/char.eo
@@ -4,4 +4,4 @@
 [data] > char
   # Converts char to string
   [] > toString
-    sprintf "%c" data > @
+    sprintf "%c" ^ > @

--- a/eo-runtime/src/main/java/org/eolang/phi/Data.java
+++ b/eo-runtime/src/main/java/org/eolang/phi/Data.java
@@ -31,6 +31,7 @@ import org.eolang.EObool;
 import org.eolang.EOfloat;
 import org.eolang.EOint;
 import org.eolang.EOstring;
+import org.eolang.EOchar;
 
 /**
  * A data container.
@@ -72,6 +73,8 @@ public interface Data<T> {
                         phi = new EOint();
                     } else if (obj instanceof String) {
                         phi = new EOstring();
+                    } else if (obj instanceof Character) {
+                        phi = new EOchar();
                     } else if (obj instanceof Double) {
                         phi = new EOfloat();
                     } else if (obj instanceof Phi[]) {

--- a/eo-runtime/src/test/eo/org/eolang/char-tests.eo
+++ b/eo-runtime/src/test/eo/org/eolang/char-tests.eo
@@ -1,0 +1,7 @@
++package org.eolang
++junit
+
+[] > charToString
+  eq. > @
+    "A"
+    'A'.toString

